event-store-plans.md

```diff
@@ -0,0 +1,174 @@
+# Event Store Improvements for V7
+
+This document is brainstorming for improvements to the Event Store functionality in Marten V7. This does not include features meant for
+"CritterStackPro" products, but does include some necessary baseline infrastructure for planned "CritterStackPro" features.
+
+In some cases, this document outlines "poor man's" subset strategies for features that we will build in a more robust (and inevitably more complicated) way 
+in "CritterStackPro"
+
+The goal is to start a discussion about what is and isn't going to be in scope for V7. When this document is "complete", the output is a new set
+of GitHub issues marked for the V7 release. 
+
+
+## Raise events from asynchronous projection updates
```

Comment:
This would be an awesome feature. This would make async projections even more usable in Marten, as having a clean way to hook into async projection updates is something I have wanted to be able to do a number of times.